packages/client/src/runGeneratorBinary.ts

```diff
@@ -1,10 +1,10 @@
 import execa from 'execa'
 
-export async function runGeneratorBinary(
-  binaryPath: string,
+export async function runGeneratorExecutable(
+  executablePath: string,
   options: any,
 ): Promise<{ stdout: string; stderr: string }> {
-  const child = execa(binaryPath)
+  const child = execa(executablePath)
```

Comment:
Here we can only execute engine binaries, right? If so, probably we can avoid renaming the function and the argument

Reply:
No, we can execute anything executable.